src/vm/wren_core.c

```diff
@@ -152,6 +153,55 @@ DEF_PRIMITIVE(fiber_error)
   RETURN_VAL(AS_FIBER(args[0])->error);
 }
 
+#define OUT_BUFSIZ 1024*16
+#define LINE_BUFSIZ 512
+DEF_PRIMITIVE(fiber_stackTrace)
+{
+  ObjFiber* fiber = AS_FIBER(args[0]);
+
+  // Allocate 16kb for our output string
+  char outputBuffer[OUT_BUFSIZ];
+  outputBuffer[0] = 0;
+  int bytesWritten = 0;
+  
+  // Allocate 512byte for one line int the output
```

Comment:
"int" the output?? "in" the output... :P

Reply:
Thanks ;)